f3.go

```diff
@@ -242,14 +242,19 @@ func (m *F3) reconfigure(ctx context.Context, manifest *manifest.Manifest) (_err
 		log.Errorw("failed to properly stop F3 while reconfiguring", "error", err)
 	}
 
-	if manifest == nil {
+	// pause if new manifest is nil
+	if manif == nil {
+		return nil
+	}
+	// pause if we are not capable of supporting this version
+	if manif.ProtocolVersion > manifest.VersionCapability {
 		return nil
```

Comment:
I'd log something (once).

Reply:
Also, maybe we should have an equality here?

Reply:
this IMO gives us more flexibility

Reply:
Eh, up to you. I guess we're unlikely to really go "backwards" anyways.